src/pem.rs

```diff
@@ -0,0 +1,220 @@
+#![allow(unused)]
```

Comment:
I'd prefer to lean on the underlying libraries for this logic. It certainly isn't necessary for OpenSSL and Security.framework, and I think schannel probably has APIs that can handle this as well?

Reply:
`schannel` and the underlying `wincrypt` APIs don't handle parsing chained PEM encoded certificates, so at least there needs to be functionality for splitting them. Beyond that, the `PrivateKey::from_pkcs8` takes DER, so either I need to convert the PEM to DER in this crate, or add that functionality to `schannel`, probably using https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-cryptstringtobinarya

Reply:
I believe CryptStringToBinary can be made to deal with PEM -> DER parsing.